vmdk_plugin/drivers/photon/photon_driver.go

```diff
@@ -156,22 +157,12 @@ func validateCreateOptions(r volume.Request, fsMap map[string]string) error {
 		r.Options[fsTypeTag] = fs.FstypeDefault
```

Comment:
As per discussion with @shaominchen and @pshahzeb, I've created issue #1538 as we want to have this in a different PR.